README.md

```diff
@@ -117,51 +117,74 @@ let make = () => {
 Using `fetchPolicy` to change interactions with the `apollo` cache, see [apollo docs](https://www.apollographql.com/docs/react/api/react-apollo/#optionsfetchpolicy).
 
 ```reason
-let (_simple, full) = UserQuery.use(~fetchPolicy=NetworkOnly, ());
+let (_simple, full) = useQuery(~fetchPolicy=NetworkOnly, (module UserQuery));
 ```
 
 Using `errorPolicy` to change how errors are handled, see [apollo docs](https://www.apollographql.com/docs/react/api/react-apollo/#optionserrorpolicy).
 
 ```reason
-let (simple, _full) = UserQuery.use(~errorPolicy=All, ());
+let (simple, _full) = useQuery(~errorPolicy=All, (module UserQuery));
 ```
 
 Using `skip` to skip query entirely, see [apollo docs](https://www.apollographql.com/docs/react/api/react-apollo/#configskip).
 
 ```reason
 let (simple, _full) =
-  UserQuery.use(
+  useQuery(
     ~skip=
-      switch (val) {
+      switch (value) {
       | None => true
       | _ => false
       },
-    (),
+    (module UserQuery),
   );
 ```
 
 ## useMutation
 
 ```reason
-module ScreamMutationConfig = [%graphql {|
+module ScreamMutation = [%graphql {|
   mutation ScreamMutation($screamLevel: Int!) {
     scream(level: $screamLevel) {
       error
     }
   }
 |}];
 
-module ScreamMutation = ReasonApolloHooks.Mutation.Make(ScreamMutationConfig);
+[@react.component]
+let make = () => {
+  /* Both variant and records available */
+  let ( screamMutation, _simple, _full ) = useMutation(~variables=ScreamMutation.makeVariables(~screamLevel=10, ()), (module ScreamMutation));
```

Comment:
Just two little thoughts:
1. I supposed that in `grapql_ppx` there is no function `makeVariables`. So I think that at the beggining in README.md it should be information to install `@baransu/graphql_ppx_re` instead of `grapql_ppx`.
2. If we assume that user should install `@baransu/graphql_ppx_re` by himself, it probably should be in peerDependencies in package.json

Reply:
Yes! The readme needs an update :)